shell/platform/darwin/ios/framework/Headers/FlutterEngine.h

```diff
@@ -0,0 +1,172 @@
+ * Callers must use `-[FlutterEngine initWithName:project:]`.
+ */
+- (instancetype)init NS_UNAVAILABLE;
+
+/**
+ * Runs a Dart program on an Isolate from the main Dart library (i.e. the library that
+ * contains `main()`).
+ *
+ * The first call to this method will create a new Isolate. Subsequent calls will return
+ * immediately.
+ *
+ * @param entrypoint The name of a top-level function from the same Dart
+ *   library that contains the app's main() function.  If this is nil, it will
+ *   default to `main()`.  If it is not the app's main() function, that function
+ *   must be decorated with `@pragma(vm:entry-point)` to ensure the method is not
+ *   tree-shaken by the Dart compiler.
+ * @return YES if the call succeeds in creating and running a Flutter Engine instance; NO otherwise.
+ */
+- (bool)runWithEntrypoint:(NSString*)entrypoint;
+
+/**
+ * Runs a Dart program on an Isolate using the specified entrypoint and Dart library,
+ * which may not be the same as the library containing the Dart program's `main()` function.
+ *
+ * The first call to this method will create a new Isolate. Subsequent calls will return
+ * immediately.
+ *
+ * @param entrypoint The name of a top-level function from a Dart library.  If nil, this will
+ *   default to `main()`.  If it is not the app's main() function, that function
+ *   must be decorated with `@pragma(vm:entry-point)` to ensure the method is not
+ *   tree-shaken by the Dart compiler.
+ * @param libraryUri The URI of the Dart library which contains the entrypoint method.  IF nil,
+ *   this will default to the same library as the `main()` function in the Dart program.
+ * @return YES if the call succeeds in creating and running a Flutter Engine instance; NO otherwise.
+ */
+- (bool)runWithEntrypointAndLibraryUri:(NSString*)entrypoint libraryUri:(NSString*)uri;
+
+/**
+ * Sets the `FlutterViewController` for this instance.  The FlutterEngine must be
+ * running (e.g. a successful call to `-runWithEntrypoint:` or `-runWithEntrypointAndLibraryUri`)
+ * before calling this method. Callers may pass nil to remove the viewController
+ * and have the engine run headless in the current process.
+ *
+ * A FlutterEngine can only have one `FlutterViewController` at a time. If there is
+ * already a `FlutterViewController` associated with this instance, this method will replace
+ * the engine's current viewController with the newly specified one.
+ *
+ * Setting the viewController will signal the engine to start animations and drawing, and unsetting
+ * it will signal the engine to stop animations and drawing.  However, neither will impact the state
+ * of the Dart program's execution.
+ *
+ * @param viewController The `FlutterViewController` (or nil) to set. Kept as a weak reference.
+ */
+- (void)setViewController:(FlutterViewController*)viewController;
+/**
+ * Gets the `FlutterViewController` (may be nil) currently used by this instance.
+ */
+- (FlutterViewController*)getViewController;
```

Comment:
`@property (nonatomic, readonly) viewController`. Objective-C getters dont have the "get" prefix.